### ISSUE TITLE
For Series I do not see watched Episodes updated.  Is this an error or not possible to implement.

### ISSUE THREAD

**Issue by rikardfries** [emilsvennesson/kodi-viaplay#45]
For Series I do not see watched Episodes updated.  Is this an error or not possible to implement.

**Comment by putrid1** [comment]
Any insight on this? Would love to see episodes as watched ✅

**Comment by Mariusz89B** [comment]
Should be fixed here:
https://github.com/Mariusz89B/kodi-viaplay/releases/tag/2.1.4

**Comment by putrid1** [comment]
> Should be fixed here:
> https://github.com/Mariusz89B/kodi-viaplay/releases/tag/2.1.4

Not sure.. 
what I'm missing is that little "watched" (hotkey W) icon after the episode name.

I appreciate your update, but I don't see that it solves this particular issue? Thanks

**Comment by Mariusz89B** [comment]
> > Should be fixed here:
> > https://github.com/Mariusz89B/kodi-viaplay/releases/tag/2.1.4
> 
> Not sure..
> what I'm missing is that little "watched" (hotkey W) icon after the episode name.
> 
> I appreciate your update, but I don't see that it solves this particular issue? Thanks

Sorry, thought you mean the category watched was missing.

**Comment by Mariusz89B** [comment]
Well, it should be possible to implement a cache list function and add an icon to watched episodes. But do not Kodi possibly have this function built-in?

**Comment by putrid1** [comment]
> Well, it should be possible to implement a cache list function and add an icon to watched episodes. But do not Kodi possibly have this function built-in?

Yes it has. And it works on everything but the viaplay-plugin. I thought it was a bug in the plugin. Or maybe it's something wrong on my end?

**Comment by Mariusz89B** [comment]
@putrid1 Yes, it seems to be some sort of issue with the listitem object. I'm not sure how to resolve this as I'm not that familiar with how plugins implement listitem objects as I only created scripts before.

**Comment by putrid1** [comment]
I dont know if this helps. I'm comparing another kodi-addon (https://github.com/CaptainTyphoo/xbmc-addon-nrk/tree/python3) and **Mark as watched**

**NRK Addon**
```
2021-11-20 19:41:52.822 T:47653    INFO <general>: initializing python engine.
2021-11-20 19:41:54.219 T:47653    INFO <general>: CPythonInvoker(44, /home/kodi/.kodi/addons/plugin.video.nrk/main.py): script successfully run
2021-11-20 19:41:54.380 T:47653    INFO <general>: Python interpreter stopped2021-11-20 19:41:52.822 T:47653    INFO <general>: initializing python engine.
2021-11-20 19:41:54.219 T:47653    INFO <general>: CPythonInvoker(44, /home/kodi/.kodi/addons/plugin.video.nrk/main.py): script successfully run
2021-11-20 19:41:54.380 T:47653    INFO <general>: Python interpreter stopped

```

**Viaplay Addon**
```
2021-11-20 19:43:43.295 T:47686    INFO <general>: initializing python engine.
2021-11-20 19:43:44.369 T:47686   ERROR <general>: NEWADDON Unknown Video Info Key "list_title"
2021-11-20 19:43:44.471 T:47686    INFO <general>: Skipped 7 duplicate messages..
2021-11-20 19:43:44.471 T:47686    INFO <general>: CPythonInvoker(48, /home/kodi/.kodi/addons/plugin.video.viaplay/default.py): script successfully run
2021-11-20 19:43:44.700 T:47686    INFO <general>: Python interpreter stopped

```
![image](https://user-images.githubusercontent.com/76736133/142737786-1fe9a97b-0d54-4d5c-b102-9952dc17301c.png)

**Comment by Mariusz89B** [comment]
@putrid1 Good find! This issue seems releated to "list_title" that is included in the listitem object, I will try to resolve it.

**Comment by Mariusz89B** [comment]
@putrid1 I removed the "list_title" error, but unfortunately it seems not to be related to marked as watched issue.

**Comment by putrid1** [comment]
@Mariusz89B Too bad.. Did you upload that last fix? I'll let you know if I find anything else suspicious
